### ISSUE TITLE
Trouble implementing AdaIn

### ISSUE THREAD

**Issue by akhauriyash** [xunhuang1995/AdaIN-style#28]
Hello. I want to implement this in Python - Tensorflow. A really great paper.
I read the paper on Adaptive instance normalization, and completely understand the decoder training. However, I am having trouble understanding how the AdaIn part of the algorithm would be **written in code**. I do not understand Lua and thus do not want to risk misunderstanding the concept.
Can someone explain the AdaIn part of their algorithm to me in pseudocode/english so that I can try to implement it? As far as I am aware, the AdaIn component only takes in the **VGG encoder feature maps for the Image and style.**

Thank you!

**Comment by diggerdu** [comment]
https://github.com/diggerdu/Vincent
https://github.com/eridgd/AdaIN-TF

**Comment by akhauriyash** [comment]
Thank you!
